thredds_crawler/meta.yaml

```diff
@@ -19,6 +20,7 @@ requirements:
         - python
         - requests
         - lxml
+        - netCDF4
```

Comment:
I'll fix. Question... is this the right place to put `netcdf4`? It's an optional dependency for one specific part of the code (and one test requires it).